src/view/com/pager/TabBar.tsx

```diff
@@ -117,6 +119,13 @@ export function TabBar({
     [],
   )
 
+  const feedsDefault = {
+    ["Following"]: _(msg`Following`),
+    // ["Discover"]: _(msg`Discover`),
+    // ["What's Hot Classic"]: _(msg`What's Hot Classic`),
+    // ["Popular With Friends"]: _(msg`Popular With Friends`),
+  } as any;
```

Comment:
@haileyok @mozzius What do yall think of this? We had some other folks feeling frustrated these aren't translateable. I'm not against the idea. Also need to check in on this code, if it does the job right for us.

Reply:
we should probably limit it to the ones we run? but other than that it's certainly not the worst idea, my only concern is it wouldn't be consistent across the app - you'd see a different name in the feeds screen, and if we did the same fix there then you still wouldn't be able to search for them